### PR TITLE
Fix line edit button

### PR DESCRIPTION
**Pull request by miltonbsn**
# Summary 

When I upgraded joint.js, I also updated joint.css and unfortunately I had made some customizations in that file. I don't think I found all but here comes a needed one.

# Change List

- Add a css customization to show hiden button on joint line
- Fix a checkbok input type 
- Remove lost character on file  

# Screenshots

<table>
<thead>
<th>Before</th>
<th>After</th>
</thead>
<tbody>
<tr>
<td>

image

</td>

<td>

image

</td>

</tr>
</tbody>
</table>

<table>
<thead>
<th>Before</th>
<th>After</th>
</thead>
<tbody>
<tr>

<td>

image

</td>

<td>

image

</td>
</tr>
</tbody>
</table>

### CODE REVIEW

File: app/sass/joint.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+///////////////////////////////////////////////////////////////////////////////
+// Joint customizations
+///////////////////////////////////////////////////////////////////////////////
+.link-tools .tool-options {
+  display: block;
```
Comment (idmarjr) [inline app/sass/joint.scss]: Out of curiosity. Is this the best way to show this `.tools-options`?
Shouldn't visible/invisible state be managed via JS?

File: app/sass/app.scss [hunk 2]
```diff
@@ -9,4 +9,5 @@
 @import 'atomic.scss';
 @import 'selectOptions.scss';
 @import 'sql.scss';
-@import 'print.scss';
\ No newline at end of file
+@import 'print.scss';
+@import 'joint.scss';
```
Comment (idmarjr) [inline app/sass/app.scss]: Can we rename it to `joint-custom.scss`?
This way we can make it clear this file is not part of _joint core_

File: app/sass/joint.scss (hunk 1 above)
Comment (miltonbsn) [reply app/sass/joint.scss]: I tried all the ways to activate this by JS but unfortunately I didn't succeed without having to do a major refactoring. For now, this is the simplest way to go :(

File: app/sass/app.scss (hunk 2 above)
Comment (miltonbsn) [reply app/sass/app.scss]: Done!

**Review by idmarjr — APPROVED** [review]
Go and Fly!
Fly baby! Fly!

**Pull request merged**